src/global.ts

```diff
@@ -1,10 +1,8 @@
-import { Prism } from './core/prism';
+import globalPrism, { Prism } from './core/prism-class';
 
-const globalSymbol = Symbol.for('Prism global');
-
-// eslint-disable-next-line no-undef
-const namespace = globalThis as Partial<Record<typeof globalSymbol, Prism>>;
-const globalPrism = (namespace[globalSymbol] ??= new Prism());
+declare global {
+	var Prism: Prism | undefined
+}
```

Comment:
```suggestion
	const Prism: Prism | undefined;
```

Is there a reason for using `var` I might miss?

Reply:
Nope, that's just what the code example I found for extending `global` was using 😛 But this suggestion is also on the wrong line.